src/components/SettingsDialog/Participants/Participants.tsx

```diff
@@ -18,12 +18,27 @@ export const Participants = () => {
   const debouncedQueryString = useDebounce(queryString);
   const [permissionFilter, setPermissionFilter] = useState<"ALL" | "OWNER" | "MODERATOR" | "PARTICIPANT">("ALL");
   const [onlineFilter, setOnlineFilter] = useState<boolean>(true);
+  const [isScrollable, setIsScrollable] = useState<boolean>(false);
+  const scrollRef = useRef<HTMLDivElement>(null);
 
   const isModerator = useAppSelector((state) => state.participants!.self.role === "OWNER" || state.participants!.self.role === "MODERATOR");
   const self = useAppSelector((state) => state.participants!.self);
   const participants = useAppSelector((state) => [state.participants!.self, ...(state.participants?.others ?? [])]);
   const existsAtLeastOneReadyUser = participants.some((p) => p.ready);
 
+  useEffect(() => {
+    const scrollableDiv = scrollRef.current;
+    if (!scrollableDiv) {
+      return undefined;
+    }
+    const mutationObserver = new MutationObserver(() => {
```

Comment:
Do we really need a useEffect and Observer for this? Can't we just use `overflow-x: hidden` and `overflow-y: scroll`?

Reply:
blub